src/Analysis/Ast/Impl/Indexing/ISymbolIndex.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Python.Parsing.Ast;
+
+namespace Microsoft.Python.Analysis.Indexing {
+    internal interface ISymbolIndex {
+        void UpdateIndex(Uri uri, PythonAst pythonAst);
+        bool isNotEmpty();
```

Comment:
Looks like a property to me. `IsEmpty { get; }` is a typical implementation (avoid Not unless really needed).